Sources/Interpreter/Interpreter.swift

```diff
@@ -296,6 +299,54 @@ public struct Interpreter {
     topOfStack.registers[i] as? Address
   }
 
+  /// Returns the value of `x` if it has a address, or `nil` if it does not.
+  func address(_ x: Operand) -> Address? {
+    switch x {
+    case .register(let instruction):
+      return topOfStack.registers[instruction] as? Address
+    case .parameter(_, let i):
+      return topOfStack.parameters[i]
+    case .constant:
+      return nil
```

Comment:
```suggestion
    case .register(let instruction):
      topOfStack.registers[instruction] as? Address
    case .parameter(_, let i):
      topOfStack.parameters[i]
    case .constant:
      nil
```

switch is an expression-statement in Swift these days